src/controllers/usersController.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const userService = require('../services/users');
+
+const getAll = (req, res) => {
+  const users = userService.getUsers();
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const userId = Number(req.params.userId);
+
+  if (isNaN(userId)) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const foundUser = userService.getUser(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundUser);
+};
+
+const create = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
```

Comment:
Check for the type of 'name' to make sure it's a string before checking if it's empty.